### PR TITLE
[T3A2][W09-A1]Johann Wong Jun Guo

### PR DESCRIPTION
**Pull request by Jslyfer**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/commands/CommandResult.java [hunk 1]
```diff
@@ -11,7 +11,7 @@
 public class CommandResult {
 
     /** The feedback message to be shown to the user. Contains a description of the execution result */
-    public final String feedbackToUser;
+    private final String feedbackToUser;
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/commands/CommandResult.java]: Think why the author use `final`.

File: src/seedu/addressbook/commands/CommandResult.java [hunk 2]
```diff
@@ -33,4 +33,11 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
         return Optional.ofNullable(relevantPersons);
     }
 
+	/**
+	 * @return the feedbackToUser
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/commands/CommandResult.java]: If the method is so simple, it's better to write method description then use `@return`, e.g. `Returns ...`. Refer to #188 .

File: src/seedu/addressbook/commands/CommandResult.java [hunk 3]
```diff
@@ -33,4 +33,11 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
...
+	 */
+	public String getFeedbackToUser() {
+		return feedbackToUser;
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/commands/CommandResult.java]: Please fix the indentation and change the tab to spaces in Eclipse, see [here](http://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs)

**Comment by jeffryhartanto** [comment]
@Jslyfer Some comments added. Please close the PR after reading comments.

File: src/seedu/addressbook/commands/CommandResult.java (hunk 1 above)
Comment (Jslyfer) [inline src/seedu/addressbook/commands/CommandResult.java]: Is it to cast the variable as an immutable object? As it is not designed for inheritance so we prohibit it?

**Pull request closed without merging**